src/Components/Alert/Hooks/useAlertContext.tsx

```diff
@@ -22,11 +28,23 @@ type Settings = {
   push: boolean
 }
 
+type Label = {
+  displayValue: string
+  field: string
+  value: string
+}
+
+type Preview = {
+  labels?: (Label | null)[]
+  displayName: string
+}
+
 export type State = {
   settings: Settings
   criteria: SearchCriteriaAttributes
   searchCriteriaID?: string
   currentArtworkID?: string
+  preview: Preview
```

Comment:
**suggestion / non-blocking**: Instead of defining new types, would it make sense to have this be of type `PreviewSavedSearch` as exposed via the Metaphysics schema? Might also make this nullable so that it can unset initially but then set as soon as we get the first MP response.

Reply:
Good idea! I could extract the type like this:

```
type PreviewSavedSearch = NonNullable<
  useAlertContextPreviewQuery["response"]["viewer"]
>["previewSavedSearch"]
```

Reply:
Nice! That looks good to me 👍🏼